base.php

```diff
@@ -2530,6 +2531,7 @@ function($level,$text,$file,$line) {
 			'PACKAGE'=>self::PACKAGE,
 			'PARAMS'=>[],
 			'PATH'=>$path,
+            'PATH_SLASH_CLEAN'=>TRUE,
```

Comment:
When this gets accepted, make all indentations consistent and don't forget to update the documentation. :-)

Reply:
Thanks, I'll check it. Phpstorm shows normal indents. 
Where should I update the docs?

Reply:
The docs are [here](https://github.com/f3-factory/F3com-data), but I would wait until this change got accepted.

Reply:
Of course, just asking